ddl/ddl.go

```diff
@@ -138,7 +138,7 @@ var (
 	// ErrCantDropFieldOrKey returns for dropping a non-existent field or key.
 	ErrCantDropFieldOrKey = terror.ClassDDL.New(codeCantDropFieldOrKey, "can't drop field; check that column/key exists")
 	// ErrInvalidOnUpdate returns for invalid ON UPDATE clause.
-	ErrInvalidOnUpdate = terror.ClassDDL.New(codeInvalidOnUpdate, "invalid ON UPDATE clause for the column")
+	ErrInvalidOnUpdate = terror.ClassDDL.New(codeInvalidOnUpdate, "Invalid ON UPDATE clause for '%s' column")
```

Comment:
https://github.com/pingcap/tidb/blob/3db7594fa53c9e2dfb71f186231d3723a817b384/mysql/errname.go#L312
here is the error message, I think we can use it directly.